packages/contracts/scripts/deploy/ubiquistick/SimpleBond.ts

```diff
@@ -23,7 +23,7 @@ const func = async (params: DeployFuncParam) => {
         throw new Error(`Unsupported network: ${args.network} Please configure it out first`);
     }
     // If testenv is true, it means that the ownership is transferred in the deployment step
-    // Must be careful when you deploy contracts to the mainnets.
+    // Must be careful when you deploy contracts to the mainnet(s).
```

Comment:
Ah, i left the `s` because i figured you would want to expand to other evm networks that are not testnets. For instance, if you consider polygon to be an L2, that is still a mainnet since people generally consider those real funds.

Reply:
We don't plan to support sidechains.

Reply:
Lets keep `mainnet`